atom/browser/api/atom_api_web_contents.cc

```diff
@@ -455,7 +456,6 @@ WebContents::WebContents(v8::Isolate* isolate,
       guest_delegate_(nullptr),
       weak_ptr_factory_(this) {
   if (type == REMOTE) {
-    guest_delegate_ = brave::TabViewGuest::FromWebContents(web_contents);
```

Comment:
When will `guest_delegate_` have non null value? It seems like it will always be null. If that is intended, can we just remove all `guest_delegate_->...` which will not be ran anyway?

Reply:
right, in `TabViewGuest::DidInitialize`